Watchman.DomainModel/Responses/Response.cs

```diff
@@ -38,5 +38,11 @@ public void SetMessage(string message)
             this.Message = message;
             this.Update();
         }
+
+        public void UpdateAvailableVariables(string[] newAvailableVariables)
+        {
+            this.AvailableVariables = newAvailableVariables;
```

Comment:
w sumie to tu też możesz dać sprawdzanie tak jak w pozostałych metodach czy wszystkie newAvailableVariables są takie same
raczej ta tablica ma kilka elementów więc nie będzie to żadnym wydajnościowym problemem